docs/data/joy/getting-started/tutorial/tutorial.md

```diff
@@ -238,7 +243,6 @@ Here's a recap of the components used:
 - [Typography](/joy-ui/react-typography/)
 - [Button](/joy-ui/react-button/)
 - [Link](/joy-ui/react-link/)
```

Comment:
```suggestion
- [Link](/joy-ui/react-link/)
- [Form Control](/joy-ui/react-form-control/)
- [Form Label](/joy-ui/react-form-label/)
- [Input](/joy-ui/react-input/)
```

This would be great, but we do not have documentation pages for either of Form Control, Form Label or Input.

Reply:
I think we can leave `FormControl` and `FormLabel` out of this PR.

Reply:
Agree